flaml/automl/automl.py

```diff
@@ -994,7 +997,10 @@ def predict(
             )
             return None
         X = self._preprocess(X)
-        y_pred = estimator.predict(X, **pred_kwargs)
+        try:
+            y_pred = estimator.predict(X, **pred_kwargs)
+        except ZeroDivisionError:
+            y_pred = 0
```

Comment:
I don't think this is the right place to catch the error because the error is specific to transformers.